### DISCUSSION TITLE
How to train a GNN on some graphs so enabling the model to predict missing links on the graphs on test part

### DISCUSSION THREAD

**Discussion by harirRzi** [pyg-team/pytorch_geometric#9345] [Q&A]
Hello. I have multiple graphs having different number of nodes which are type of two classes. I want to train a GNN model to train on some of those graphs and then on some other part as a test to predict missing links on each of the graphs separately in test part. Is there any example for it to help me?

**Comment by rusty1s** [comment]
Prediction of new/unseen graphs is easily doable with GNNs, see, e.g., `examples/ppi.py` or `examples/mutag_gin.py`. For link prediction, we don't have a multi-graph example though, but you should be able to simply apply your trained model on unseen graphs and perform link prediction on top.
